### PR TITLE
Migrated Container portlet from hibernate to SQL

### PR DESCRIPTION
**Pull request by hassan-mustafa-baig**
### Proposed Changes
* Data access layer for the container portlet has been migrated from Hibernate to SQL. This has fixed the issue https://github.com/dotCMS/core/issues/23454.

### Checklist
- [X] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/curl-test/BringBack.postman_collection.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "07fa881b-476f-49a9-9e00-997cc61e4840",
+		"_postman_id": "bcb92a0a-19ee-4710-8120-847372062cac",
```
Comment (jcastro-dotcms) [inline dotCMS/src/curl-test/BringBack.postman_collection.json:3]: Revert this change, as it is not necessary.

File: dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPIImpl.java [hunk 2]
```diff
@@ -722,27 +720,27 @@ public Container save(final Container container,final List<ContainerStructure> c
 		boolean existingInode=false;
 
 		if(UtilMethods.isSet(container.getInode())) {
-            try {
```
Comment (jcastro-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPIImpl.java:725]: Revert all the extra blank spaces in order to keep the PR as clean as possible.

File: dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerFactory.java [hunk 3]
```diff
@@ -26,14 +26,14 @@ public interface ContainerFactory {
 	 */
 	void save(Container container) throws DotDataException;
 
-    /**
-     * Save an existing container
-     * @param container
-     * @param existingId
-     * @throws DotDataException
-     */
+	/**
```
Comment (jcastro-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerFactory.java:29]: Revert unnecessary blank spaces here too.

**Review by jcastro-dotcms — CHANGES_REQUESTED** [review]

File: dotCMS/src/curl-test/BringBack.postman_collection.json (hunk 1 above)
Comment (hassan-mustafa-baig) [reply dotCMS/src/curl-test/BringBack.postman_collection.json]: This has been addressed

File: dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerFactory.java (hunk 3 above)
Comment (hassan-mustafa-baig) [reply dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerFactory.java:28]: This has been addressed

**Review by jcastro-dotcms — APPROVED** [review]
